HIP-7.md

```diff
@@ -0,0 +1,113 @@
+| hip | title                                           | status | author     | created    |
+| --- | ----------------------------------------------- | ------ | ---------- | ---------- |
+| 7   | Draft multisig transactions on long-tail chains | Draft  | aroralanuk | 2024-01-10 |
+
+### **Brief Summary / Abstract**
+
+> "Uh, open up the safe, bitches got a lot to say" - Iggy Azalea
```

Comment:
This is a family friendly open source community. Please consider using a different quote.